lib/dangermattic/plugins/android_strings_checks.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Danger
+  # Plugin for checking the strings.xml localization files for Android.
+  #
+  # @example Check if a strings.xml file refers to another string resource
+  #
+  #          android_strings_checks.check_strings_do_not_refer_resource
+  #
+  # @see Automattic/dangermattic
+  # @tags android, localization
+  #
+  class AndroidStringsChecks < Plugin
+    # Check if translatable strings reference another string resource in 'strings.xml' files in a pull request.
+    #
+    # @return [void]
+    def check_strings_do_not_refer_resource
+      warning_message = "This PR adds a translatable entry which references another string resource; this usually causes issues with translations.\n" \
+                        'Please make sure to set the `translatable="false"` attribute:'
+
+      git_utils.check_added_diff_lines(
```

Comment:
`git_utils` is available thanks to the change done in `danger_plugin.rb` above.

I wonder what would happen if someone used this "sub-plugin" directly?

Reply:
I think that shouldn't be a problem, and is actually an upside. I already see a few other utilities that can be moved to `git_utils` so that they don't need to be repeated across several `Dangerfile`s.

Reply:
> I already see a few other utilities that can be moved to `git_utils` so that they don't need to be repeated across several `Dangerfile`s.

Sure 👍 

Would the import work though, if one fetches the sub-plugin only instead of the whole Dangermattic?

Reply:
Once you add the `Dangermattic` Gem, all plugins are instantiated and made available -- it's all magic from Danger 🙂.
It is indeed possible to fetch only a certain plugin in isolation, but then I **think** it would fail 🤔